### ISSUE TITLE
axis.reset() gives error when using add_line(interactive=True) [BUG]

### ISSUE THREAD

**Issue by itzzp** [highfestiva/finplot#343]
### Requirements (place an `x` in each of the `[ ]`)**
* [x] I realize finplot is not a web lib. (Hint: it's native!)
* [x] I've read the [snippets](https://github.com/highfestiva/finplot/wiki/Snippets) and not found what I'm looking for.
* [x] I've searched for any related issues and avoided creating a duplicate issue.
* [x] I've updated finplot (`pip install -U finplot`).
* [x] I've supplied the required data to run my code below.

### Code to reproduce

```python
#!/usr/bin/env python3

import finplot as fplt
from functools import lru_cache
from PyQt5.QtWidgets import QApplication, QGridLayout, QMainWindow, QGraphicsView, QComboBox, QLabel
from pyqtgraph.dockarea import DockArea, Dock
from threading import Thread
import yfinance as yf

app = QApplication([])
win = QMainWindow()
area = DockArea()
win.setCentralWidget(area)
win.resize(1600,800)
win.setWindowTitle("Docking charts example for finplot")

# Set width/height of QSplitter
win.setStyleSheet("QSplitter { width : 20px; height : 20px; }")

# Create docks
dock_0 = Dock("dock_0", size = (1000, 100), closable = True)
dock_1 = Dock("dock_1", size = (1000, 100), closable = True)
dock_2 = Dock("dock_2", size = (1000, 100), closable = True)
area.addDock(dock_0)
area.addDock(dock_1)
area.addDock(dock_2)

# Create example charts
combo = QComboBox()
combo.setEditable(True)
[combo.addItem(i) for i in "AMRK FB GFN REVG TSLA TWTR WMT CT=F GC=F ^FTSE ^N225 EURUSD=X ETH-USD".split()]
dock_0.addWidget(combo, 0, 0, 1, 1)
info = QLabel()
dock_0.addWidget(info, 0, 1, 1, 1)

# Chart for dock_0
ax0,ax1,ax2 = fplt.create_plot_widget(master=area, rows=3, init_zoom_periods=100)
area.axs = [ax0, ax1, ax2]
dock_0.addWidget(ax0.ax_widget, 1, 0, 1, 2)
dock_1.addWidget(ax1.ax_widget, 1, 0, 1, 2)
dock_2.addWidget(ax2.ax_widget, 1, 0, 1, 2)

# Link x-axis
ax1.setXLink(ax0)
ax2.setXLink(ax0)
win.axs = [ax0,ax1,ax2]

@lru_cache(maxsize = 15)
def download(symbol):
    return yf.download(symbol, "2020-01-01")

@lru_cache(maxsize = 100)
def get_name(symbol):
    return yf.Ticker(symbol).info ["shortName"]

def update(txt):
    df = download(txt)
    if len(df) < 20: # symbol does not exist
        return
    info.setText("Loading symbol name...")
    price = df ["Open Close High Low".split()]
    ma20 = df.Close.rolling(20).mean()
    ma50 = df.Close.rolling(50).mean()
    volume = df ["Open Close Volume".split()]
    ax0.reset() # remove previous plots
    ax1.reset() # remove previous plots
    ax2.reset() # remove previous plots
    fplt.candlestick_ochl(price, ax = ax0)
    fplt.plot(ma20, legend = "MA-20", ax = ax1)
    fplt.plot(ma50, legend = "MA-50", ax = ax1)
    fplt.volume_ocv(volume, ax = ax2)


    # interactive=True
    fplt.add_line(p0=(df.index[-10], df['Close'].iloc[-10]), p1=(df.index[-1], df['Close'].iloc[-1]), ax=ax0, width=4, interactive=True)  # this DOES NOT work and gives an error when selecting a different symbol using the dropdownlist


   # interactive=False
 #  fplt.add_line(p0=(df.index[-10], df['Close'].iloc[-10]), p1=(df.index[-1], df['Close'].iloc[-1]), ax=ax0, width=4, interactive=False)  # this DOES work when selecting a different symbol using the dropdownlist


    fplt.refresh() # refresh autoscaling when all plots complete
    Thread(target=lambda: info.setText(get_name(txt))).start() # slow, so use thread

combo.currentTextChanged.connect(update)
update(combo.currentText())

fplt.show(qt_exec = False) # prepares plots when they're all setup
win.show()
app.exec_()
```

### Describe the bug

The above code is the example code 'docklable.py' with an added line:
fplt.add_line(p0=(df.index[-10], df['Close'].iloc[-10]), p1=(df.index[-1], df['Close'].iloc[-1]), ax=ax0, width=4, interactive=True)

When a different symbol is selected (by using the dropdown-list) the update function is triggered:: 
update() executes ax0.reset()
ax0.reset() executes finplot.FinPolyLine.removeSegement(self, seg)
finplot.FinPolyLine.removeSegment(self, seg) executes self.scene().removeItem(seg) 
but FinPolyLine.scene() returns a None, which does not have the attribute 'removeItem' (see screenshot below)

### Expected behavior
The expected behavior is that the axis is reset and the data for the selected symbol is loaded and shown. Like when a line would be added with the parameter interactive=False

#### Screenshots

![image](https://user-images.githubusercontent.com/773814/172618731-8eafa517-ca7b-4c54-9fe0-ae90fb3f8b71.png)

#### Reproducible in:

*OS*: ubuntu
*finplot version*: latest, commit 8dde95f
*pyqtgraph version*: 0.12.4
*pyqt version*: PyQt5   5.15.6

**Comment by highfestiva** [comment]
Fix available in 6e36fe0.

**Comment by itzzp** [comment]
Thanks!
